docs/examples/rrlyr-period.ipynb

```diff
@@ -0,0 +1,134 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.    # Plot folded light curve
</pre><p>It may be beneficial to break the plotting code up into a different cell, just for clarity.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/tape/pull/235/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>